config/example/config.txt

```diff
@@ -77,23 +77,25 @@ KICK_INACTIVE
 ##
 ## default probablity is 1, increase to make that mode more likely to be picked
 ## set to 0 to disable that mode
-PROBABILITY EXTENDED 2
-PROBABILITY NUCLEAR 3
-PROBABILITY ABDUCTION 0
-PROBABILITY CHANGELING 3
-PROBABILITY CULT 4
-PROBABILITY EXTEND-A-TRAITORMONGOUS 5
-PROBABILITY TRAITORCHAN 4
-PROBABILITY TRAITORVAMP 4
-PROBABILITY BLOB 2
+PROBABILITY EXTEND-A-TRAITORMONGOUS 3
+PROBABILITY TRAITORCHAN 3
+PROBABILITY TRAITORVAMP 3
+PROBABILITY NUCLEAR 2
 PROBABILITY REVOLUTION 0
-PROBABILITY HEIST 1
+PROBABILITY SHADOWLING 2
+PROBABILITY CULT 4
+PROBABILITY CHANGELING 3
 PROBABILITY WIZARD 2
-PROBABILITY VAMPIRE 3
+PROBABILITY MALFUNCTION 2
+PROBABILITY BLOB 1
 PROBABILITY RAGINMAGES 0
 PROBABILITY METEOR 0
-PROBABILITY TRAITOR 0
-PROBABILITY SHADOWLING 2
+PROBABILITY EXTENDED 1
+PROBABILITY HEIST 0
+PROBABILITY VAMPIRE 3
+PROBABILITY BORER 0
+PROBABILITY XENOS 0
+PROBABILITY MUTINY 0
```

Comment:
Borer, Xenos,  Malfunction, and Mutiny should all be removed as they're not modes anymore.